crates/sui-types/src/effects/mod.rs

```diff
@@ -425,6 +425,7 @@ pub trait TransactionEffectsAPI {
     fn unsafe_add_object_tombstone_for_testing(&mut self, obj_ref: ObjectRef);
 }
 
+#[derive(Clone, Copy)]
```

Comment:
Clone is fine for this, but we shouldn't have `Copy` -- this struct is not a trivial size, it would be good to know when you're about to copy it.